plugins/dell/fu-plugin-dell.c

```diff
@@ -749,8 +750,8 @@ fu_plugin_dell_detect_tpm (FuPlugin *plugin, GError **error)
 					    "Updating disabled due to TPM ownership");
 	}
 	/* build GUIDs from TSS strings */
-	if (!fu_plugin_dell_add_tpm_model (dev, error))
-		return FALSE;
+	if (!fu_plugin_dell_add_tpm_model (dev, &error_tss))
+		g_debug ("Couldn't build instances: %s", error_tss->message);
```

Comment:
lower case, e.g. `could not build instance IDs` pls

Reply:
fixed.  Did this fix your problem then?

Reply:
Gahh, no, still:

    # FuPluginDell-DEBUG: TPM Status: 0x101
    # FuPluginDell-DEBUG: Creating primary TPM GUID 0000-1.2 and secondary TPM GUID 0000-2.0
    # FuDevice-DEBUG: using 3ca929adb22a017654a24ce341b35205d2ec45cd for DELL-c8926cae-cee4-5d3b-80e5-70bd5ff1c101lu
    Bail out! FATAL-WARNING: Failed to create connection with service: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name is not activatable
    --- stderr ---
    ERROR:tcti:src/tss2-tcti/tcti-device.c:439:Tss2_Tcti_Device_Init() Failed to open device file /dev/tpm0: Permission denied 
    WARNING:tcti:src/tss2-tcti/tctildr.c:62:tcti_from_init() TCTI init for function 0x7effb4a3ffc0 failed with a000a 
    WARNING:tcti:src/tss2-tcti/tctildr.c:92:tcti_from_info() Could not initialize TCTI named: tcti-device 
    ERROR:tcti:src/tss2-tcti/tctildr-dl.c:150:tcti_from_file() Could not initialize TCTI file: libtss2-tcti-default.so 

    ** (/home/hughsie/Code/fwupd-stable/build/plugins/dell/dell-self-test:109069): WARNING **: 19:30:57.048: Failed to create connection with service: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name is not activatable

Reply:
Try as root. If it fails only as non root I think something else is going on.

Reply:
Ohh, it works indeed as root.

Reply:
I think it might be something with tpm2 abrmd then not spinning up from dbus activation

Reply:
Do you see similar failures with `uefi-self-test` as non-root? I would think so.